test/spec/modules/kargoBidAdapter_spec.js

```diff
@@ -356,4 +356,99 @@ describe('kargo adapter tests', function () {
       expect(resp).to.deep.equal(expectation);
     });
   });
+
+  describe('user sync handler', function() {
+    const clientId = '74c81cbb-7d07-46d9-be9b-68ccb291c949';
+    var shouldSimulateOutdatedBrowser,
+        uid,
+        isActuallyOutdatedBrowser;
```

Comment:
There are some extra spaces before these variables that are causing some lint errors when I run `gulp lint`.  See below for a copy of the error message:

```
/Users/jsnellbaker/git/Prebid.js/test/spec/modules/kargoBidAdapter_spec.js
  394:1  error  Expected indentation of 6 spaces but found 8  indent
  395:1  error  Expected indentation of 6 spaces but found 8  indent
```

Can you make this fix?  

Normally linting errors would be caught by circleci.  So going forward, you may want to look into updating/rebasing your fork with the latest version of master so we can catch these automatically when you submit a PR.

Reply:
Ah good catch, I'll fix and rebase as well.